content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -1,117 +1,179 @@
+Set the `node_size` based on your server's expected load and the amount of memory you can make available to `rippled`.
+
+The node size is talking about how big of an object to treat as one "node" in the ledger's underlying tree structure. It's not directly configuring storage or RAM, but it affects how much RAM your server uses and how fast your server can fetch things from disk.
+
+For example, think of node size as the size of cardboard boxes you use to ship vases from a warehouse to your store. A larger cardboard box means that you can ship more vases at once with fewer trips from the warehouse. A large node size means that you can get more `rippled` transaction data, for example, with fewer trips to disk storage.
+
+However, with a larger box, you also need to include more packing material to keep the vases from breaking during shipping, which takes up precious space in the box. Likewise, with a larger node size, you can get a lot of transaction data, but the data may also contain `rippled` start up information and ledger reports, which takes up precious RAM that can't be used for the subset of data you're actually using.
+
+A part of tuning these configurations is about choosing the right node size so that the subset of data you're actually using can be retrieved in the fewest trips to disk storage, while also ensuring that all of it, including other data you aren't using right now, can fit in your available RAM.
+
+Ripple recommends you always use the largest node size your available RAM can support. See the following table for recommended settings.
 
 #### Recommendation
 
-| Available RAM for `rippled` | `node_size` value | Notes                              |
-|:----------------------------|:------------------|:-----------------------------------|
-| < 8GB                       | `tiny`            | Not recommended                    |
-| 8GB                         | `small`           |                                    |
-| 16GB                        | `medium`          |                                    |
-| 32GB                        | `huge`            | Recommended for production servers |
+Each `node_size` has a corresponding requirement for available RAM. For example, if you set `node_size` to `huge`, you should have at least 32GB of available RAM to help ensure that `rippled` can run smoothly.
+
+To tune your server, it may be useful to start with `tiny` and increase the size to `small`, `medium`, and so on as you refine the requirements for your use case.
+
+***TODO: Small thing, but I switched the order of the node size and RAM columns because the node size column contains the value that you are setting and should be the starting point of focus.***
```

Comment:
That's funny because I thought just the opposite 😂! I thought that if you know you want to run a production server - you'd look at the node size and RAM to see what you need to provide. But I see what you are saying - which is that the user is actually thinking - I have 16GB of RAM available, so I need to set my node size to medium and I can't run a production server. I was thinking that people would make the amount of RAM they need available based on what kind of server they want to run. But I get it! Changing it back. 🙌